packages/vis/build.sh

```diff
@@ -3,15 +3,15 @@ TERMUX_PKG_DESCRIPTION="Modern, legacy free, simple yet efficient vim-like edito
 TERMUX_PKG_LICENSE="ISC"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="0.9"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=https://github.com/martanne/vis/archive/v$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=bd37ffba5535e665c1e883c25ba5f4e3307569b6d392c60f3c7d5dedd2efcfca
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="liblua53, libtermkey, lua53-lpeg, ncurses"
+TERMUX_PKG_DEPENDS="libtermkey, lua-lpeg, ncurses"
```

Comment:
Is there any reason to not add `liblua54`?

Reply:
Not really, I just assumed that if it's a dependency of a dependency, then no need for it to be mentioned (and I made a wrong assumption in my head about how it'd add to maintainability since there are less dependencies to worry about).

@sylirre did explain well why it needs to be added, in the next comment.